plugins/mcreator-localization/help/default/biome/gen_erosion.md

```diff
@@ -0,0 +1,19 @@
+Erosion controls how eroded the terrain is. Smaller values describe biomes that 
+usually generate in flatter world regions.
+
+Biomes with similar erosion will generate closer together
+and will compete for the same spot in the world when generating.
+Too similar values will result in some biomes not generating.
+
+While values from -2 to 2 are valid, vanilla biomes only use values in range
+from -1 to 1.
+
+Overworld vanilla biomes use these value ranges:
+
+* -1.0 to -0.78

```

Comment:
Could you be more specific about what biomes use these values?

Reply:
No, as those ranges are applied to different biome combinations and it is not really possible to extract the biomes that use them. You can check the code and see MC has pre-defined ranges it uses, so they are listed here.